openmc/weight_windows.py

```diff
@@ -48,6 +48,10 @@ class WeightWindows(IDManagerMixin):
     survival_ratio : float
         Ratio of the survival weight to the lower weight window bound for
         rouletting
+    max_lower_bound_ratio: float
+        Maximum allowed ratio of a weight window's lower bound to particle weight.
+        A factor will be applied to raise the weight window to the particle's weight
+        during transport if exceeded. Only applied if > 1.0. (Default: 1.0).
```

Comment:
```suggestion
    max_lower_bound_ratio: float
        Maximum allowed ratio of a particle's weight to the weight window's
	lower bound. A factor will be applied to raise the weight window to
	be lower than the particle's weight by a factor of max_lower_bound_ratio
        during transport if exceeded. Only applied if > 1.0. (Default: 1.0).
```

Reply:
flipped the meaning of the ratio here to be correct.

Reply:
Ah, yep. Thanks!